assets/js/components/user-input/UserInputApp.js

```diff
@@ -71,31 +87,24 @@ export default function UserInputApp() {
 								<Grid className="googlesitekit-user-input__header">
 									<Row>
 										<Cell
-											lgSize={ 6 }
-											mdSize={ 8 }
-											smSize={ 4 }
+											size={ 12 }
+											className="googlesitekit-user-input__question-number"
 										>
+											{ sprintf(
+												'%d / 3',
```

Comment:
This string should be translated, eg:

```suggestion
												__( '%d / 3' ),
```